dashboard/app/models/sections/section_instructor.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: section_instructors
+#
+#  id            :bigint           not null, primary key
+#  instructor_id :integer          not null
+#  section_id    :integer          not null
+#  invited_by_id :integer
+#  deleted_at    :datetime
+#  status        :integer          not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+# Indexes
+#
+#  index_section_instructors_on_deleted_at                    (deleted_at)
```

Comment:
Something that has gotten us in the past is that NULL != NULL in SQL. Do we see that being a problem? This isn't a unique index so probably not?

Reply:
ah makes sense! SGTM!